### ISSUE TITLE
Using GUI to change marker positions: new positions are not saved

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-gui#1060]
https://docs.google.com/spreadsheets/d/1E9Em79S5Y52hfAePmK-XcvTcPYwFWQBSchYemuWYTfE/edit#gid=1519816561

**Comment by aymanhab** [comment]
@chrisdembia I just loaded arm26, saved it in 4.0, changed the location of a marker and saved and the changes were saved. There must be a specific scenario to reproduce. Any details? @carmichaelong can you reproduce? Thank you

**Comment by chrisdembia** [comment]
@carmichaelong let me know if you want to work together on reproducing this.

**Comment by aymanhab** [comment]
For the record, I tested only editing the numbers in the Properties panel and not other ways (e.g. Graphically dragging or changing the Frame the marker is attached to, or loading Markers from file and editing or creating a new Marker and edit 😸 ) Would be good if a witness can help us narrow the search path.

**Comment by chrisdembia** [comment]
The issue was specifically moving the marker graphically. Sorry that wasn't clera.

**Comment by chrisdembia** [comment]
I am having some difficulty reproducing the situation.

Rachel was also saying the issue is sporadic; it doesn't always happen.

Whether or not a marker is fixed can also complicate the matter.

@rwj1 could I come by the HPL to observe the bug?

**Comment by carmichaelong** [comment]
@chrisdembia FYI I also haven't been able to reproduce yet.

**Comment by aymanhab** [comment]
@chrisdembia @carmichaelong @jenhicks I'm woking on a fix to refresh the text faster and allow undo, but it is not clear if related or if it should be in this last sprint. Let me know your thoughts.

**Comment by aymanhab** [comment]
Nice investigative work @chrisdembia 
Loading markers from a file has changed a few times including last time when we moved MarkerSet to be component, every Marker addition likely goes through the expensive initialization sequence (though 20 seconds is too much and I'm sure we can optimize). 
In step 5, do they get different behavior if they click in the Navigator or elsewhere in the application?
Is it the case that one of the 3 markers didn't move while the two others did? Just clarifying

**Comment by chrisdembia** [comment]
Lauren sent me her own files and I still could not reproduce the issue. The markers did take a long time to load on my computer.

Could the issue be related to the speed of the computer? Or Windows vs Mac?

**Comment by aymanhab** [comment]
I don't think it's the speed of the computer, or Mac vs. Window,  I'd suspect it's model having lots of muscles and wrapping objects that take forever to render/initialize. We could try to add all Merkers at once I just want to make sure we understand the scope.

**Comment by chrisdembia** [comment]
> In step 5, do they get different behavior if they click in the Navigator or elsewhere in the application?

Yes, all 3 markers move if they first click in the Properties area.

> Is it the case that one of the 3 markers didn't move while the two others did? 

Correct.

**Comment by chrisdembia** [comment]
Thanks for reviewing.

**Comment by aymanhab** [comment]
Then I think I have a fix for this (other than perfomance) to make the properties window update as soon as the user is done dragging. I just need to get my environment up again.

**Comment by chrisdembia** [comment]
Sounds good.

I could not reproduce on Windows.

**Comment by aymanhab** [comment]
If we can pass the Mac build to Lauren or Rachel to confirm that would be awesome since we couldn't reproduce on our machines. Thanks @chrisdembia 👍
